MobalMobal/MobalMobal/Account/AccountViewController.swift

```diff
@@ -103,14 +105,15 @@ class AccountViewController: UIViewController {
         verticalStackView.snp.makeConstraints { make in
             make.center.equalToSuperview()
         }
-        setLayoutFlag = true
         super.updateViewConstraints()
     }
     
     // MARK: - Actions
     @objc
     private func closeBtn() {
-        self.dismiss(animated: true, completion: nil)
+        if let presentingViewController: UIViewController = navigationController?.presentingViewController {
+            presentingViewController.dismiss(animated: true, completion: nil)
+        }
```

Comment:
아래처럼 간단하게 쓸 수 있을 것 같습니당!
```suggestion
        presentingViewController?.dismiss(animated: true, completion: nil)
```
<br/>

그리고 이건 질문인데, 아래 3가지 차이점이 뭔지 아시나요..? 결과는 다 똑같은거 같은데ㅜㅜ
`presentingViewController`를 선택한 이유가 무엇일지요?🧐

```swift
self.presentingViewController?.dismiss(animated: true, completion: nil)
self.navigationController?.dismiss(animated: true, completion: nil)
self.dismiss(animated: true, completion: nil)
```

Reply:
와.. navigationController 자체를 dismiss하면 되는군요. 이걸 어떻게 연속적으로 pop하고 마지막에 dismiss해야할지 고민하다가 구글링했는데,, 진짜 천잽니다.. 감사해여 ㅜㅜ
보여주신 코드들 중 2번째 줄, 내비게이션컨트롤러 자체를 dismiss하는게 개인적인 생각으로 더 직관적인 것 같은데 어떻게 생각하시나여?

Reply:
동감합니다!!! navigation dismiss가 좋을것 같아요